### ISSUE TITLE
[question] is source engine usable already or only goldsrc?

### ISSUE THREAD

**Issue by Heldex** [LogicAndTrick/sledge#260]
is source engine usable already or only goldsrc?
thanks

**Comment by TeamSpen210** [comment]
Sledge currently only supports goldsource.

**Comment by Heldex** [comment]
Is there an ETA on source being supported?

**Comment by LogicAndTrick** [comment]
Not at the moment unfortunately, Source *will* be supported one day, but I couldn't tell you what kind of time frame it'll be.

**Comment by CaskoChan** [comment]
Please continue this project, its the only editor that can make mapping in source engine a better experience . Please, please, please don't stop developing this. As soon as this supports source engine, im dropping hammer in the fucking trash can, and start using your tool.

**Comment by LogicAndTrick** [comment]
Thanks, I plan to continue work, but progress still be slow. My graphics code at the moment is rubbish so I'm investing time into research at the moment, which is why not much is happening in the code for now. Once I can nail down a reasonable 3D rendering approach, I should be able to make some real progress with new editing features.

**Comment by HeadClot** [comment]
@LogicAndTrick Look into BGFX if you need code for rendering stuff that is cross platform :)

**Comment by LogicAndTrick** [comment]
Thanks for the tip, it looks like a cool project. I've already made some good progress using [Veldrid](https://github.com/mellinoe/veldrid/) which seems to cover a similar range of features, so I'll stick to that for now.

**Comment by CaskoChan** [comment]
Hey, i noticed you're working on this again, great to see, i hope you can get source engine working soon

**Comment by LogicAndTrick** [comment]
Yep! I'm doing some internal testing for a few days but after that I'll be releasing a public beta for the rewrite project I've been working on. Currently it's still HL1 only but it lays all the groundwork for Source and other engines to be added easily.
